docs/source/traits_user_manual/debugging.rst

```diff
@@ -50,20 +76,21 @@ by adding a custom exception handler:
 
 (For example, you could add this to the top of the original code block.)
 
-Re-running the original code example with this custom handler will now raise
-the following traceback::
+Re-running the original code example with the exception handler will now raise
+the following traceback:
```

Comment:
Nit: `::` defaults to use Python highlighting for the block. But the output is on the console, not Python code.

Reply:
I think the Python highlighting should work fine for traceback or console stuff.

Reply:
Done reverting back to `::`. 
I kept the `code-block:: python` for the other blocks. These blocks get top and bottom borders while the ones with `::` don't.  That is to stay being consistent with the section below that I have not modified.
![Screenshot 2020-11-05 at 18 18 11](https://user-images.githubusercontent.com/3673984/98280599-8a8bbf80-1f93-11eb-86b9-014679b4e43c.png)